test/divider_test.dart

```diff
@@ -0,0 +1,428 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// divider_test.dart
+// Tests for Integer Divider
+//
+// 2024 August
+// Author: Josh Kimmel <joshua1.kimmel@intel.com>
+
+// ignore_for_file: avoid_types_on_closure_parameters
+
+import 'dart:async';
+import 'dart:collection';
+import 'dart:math';
+
+import 'package:logging/logging.dart';
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/rohd_hcl.dart';
+import 'package:rohd_vf/rohd_vf.dart';
+import 'package:test/test.dart';
+
+// helper method to compute 2's complement representation
+// of the provided value based on its bit width
+int _twosComp(int val, int bits) {
```

Comment:
the `int`s from ROHD's `LogicValue` are already always positive and truncated to the proper number of bits, so you shouldn't need to do this

Reply:
For whatever reason (I didn't do too much digging), this function is still necessary as part of my checking so leaving in for now

Reply:
ok sure